code/modules/mob/living/say.dm

```diff
@@ -94,19 +94,58 @@ var/list/channel_to_radio_key = new
 		if(S.speaking && (S.speaking.flags & NO_STUTTER || S.speaking.flags & SIGNLANG))
 			continue
 
-		if((HULK in mutations) && health >= 25 && length(S.message))
+		var/force_slurring = 0
+		var/force_shouting = 0
+		var/force_stuttering = 0
+		var/force_mute = 0
+
+		for(var/datum/modifier/M in modifiers)
+			if(M.speech_slurring)
+				force_slurring += M.speech_slurring
+			if(M.speech_stuttering)
+				force_stuttering += M.speech_stuttering
+			if(M.speech_shouting)
+				force_shouting += M.speech_shouting
+			if(M.speech_mute)
+				force_mute += M.speech_mute
+
+		if(force_slurring && force_slurring > 0)
+			force_slurring = TRUE
+		else
+			force_slurring = FALSE

```

Comment:
```
if(force_slurring < 1)
    force_slurring = FALSE
```
(And so on and so forth. If it's greater than 1 it's true, and that's the paradigm this uses)